i18n/chs/package.i18n.json

```diff
@@ -21,6 +21,7 @@
 	"cmake-tools.command.cmake.outline.cleanConfigure.title": "清除重新配置",
 	"cmake-tools.command.cmake.clean.title": "清理",
 	"cmake-tools.command.cmake.cleanRebuild.title": "清除重新生成",
+	"cmake-tools.command.cmake.openConfiguration.title": "清除重新生成",
```

Comment:
Just FYI, you don't need to edit these i18n json files.  Our automated process will take care of it.  You are welcome to leave the changes (perhaps you wanted them for your fork), but they will be overwritten when we merge this in.

Reply:
@bobbrow 
I was wondering if I should removed these from my PR. I am not quite sure what to do if I need to add a new localized string or change one. What is the proper procedure?

Reply:
You don't need to do anything for localization besides call the `localize` function in your code with a new string+id.  If you have feedback on an existing translation, we can take that back to our localizers and have them update it, but you would do that by opening a new issue rather than creating a pull request to edit the json directly.